### PR TITLE
Closes #321 disable autoplay and execute show() on ready flag

### LINKED ISSUE #321: Functionality is totally broken
I'm not sure exactly what's happened, but I think YouTube has made some API changes around CORS with the player widget.

TODO:
- [ ] Investigate & repurpose this issue

### PR DESCRIPTION
**Pull request by andy-shi88**
Disable autoplay due to the new autoplay policy on some browser.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### Motivation and Context
- [x] Since browser now don't support autoplay by default (it require user to has some action taken first), and we only execute the `show()` function on `playing` flag which is triggered by the autoplay, the player will never show.
- [x] Closes #321 
- [x] autoplay is disabled now.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change), not sure if it's count as breaking or not, but have to disable autoplay.

### Description
- [ ] disable autoplay to follow the new autoplay policy
- [ ] execute show() in `ready`

### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

File: js/everything.js [hunk 1]
```diff
@@ -137,6 +136,7 @@ var ZenPlayer = {
                 that.setupTitle();
                 that.setupVideoDescription(videoID);
                 that.setupPlyrToggle();
+                that.show();
```
Comment (shakeelmohamed) [inline js/everything.js]: Showing the player defeats the purpose of this project, can we instead start playing here? I had that working once upon a time, then reverted it once we set `autoplay=true`

**Comment by shakeelmohamed** [comment]
Thanks for getting this started @andy-shi88! 🎉

File: js/everything.js (hunk 1 above)
Comment (andy-shi88) [reply js/everything.js]: we can start playing here by calling `play`, but still we'll need the user to enable the `autoplay` on the site already to do that, we'll get `Autoplay is only allowed when approved by the user, the site is activated by the user, or media is muted.`  

one way we can do it is, we can `play` here, but in the `play` function we'll need to check if autoplay is not allowed by user, then we show the player to let user click the play button.

**Comment by andy-shi88** [comment]
@shakeelmohamed I pushed new changes that play on ready, but check in 2 second if the status is still not playing then consider the autoplay is being disabled, then show the player.

Is there any other way to detect the embedded player fail to play? in `audio` or `video` it's possible to check if `play()` function is being hold by the autoplay flag [here](https://stackoverflow.com/a/49939655/7616247) but for embedded player I can't find any, that's why I go with 2 second check instead.

**Comment by andy-shi88** [comment]
@shakeelmohamed can you check?

**Comment by shakeelmohamed** [comment]
@andy-shi88 Can we do something like show the player, play the video, then hide the player again?

**Comment by shakeelmohamed** [comment]
Closing due to inactivity.

**Pull request closed without merging**